components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Models/StandaloneCliModel.cs

```diff
@@ -56,6 +56,11 @@ public class StandaloneCliModel {
         /// </summary>
         public uint DefaultQueueSize { get; set; } = 1;
 
+        /// <summary>
+        /// set the default data change filter for monitored items
+        /// </summary>
+        public DataChangeTriggerType DefaultDataChangeFilter { get; set; } = DataChangeTriggerType.StatusValue;
```

Comment:
>DefaultDataChangeFilter [](http://example.com/codeflow?start=37&length=23)

We should have the this default configurable per environment variable. Like for example we do for the DefaultSamplingInterval